companies/moebel-kraft.json

```
@@ -0,0 +1,20 @@
+{
+    "slug": "moebel-kraft",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Möbel-Kraft Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld ",
```

```suggestion
    "address": "Ziegelstraße 1\n23795 Bad Segeberg\nDeutschland",
```
corrected address, see [https://www.moebel-kraft.de/datenschutz](https://www.moebel-kraft.de/datenschutz)